jbpm-container-test/jbpm-in-container-test/jbpm-container-test-suite/src/test/resources/org/jbpm/test/container/archive/localtransactions/tomcat-context.xml

```diff
@@ -13,8 +13,9 @@
 
     <!-- Database resources -->
     <Resource name="rawDataSource" uniqueName="rawDataSource" auth="Container" type="${org.jbpm.datasource.driver.class}" username="${org.jbpm.datasource.username}"
-              user="${org.jbpm.datasource.username}" password="${org.jbpm.datasource.password}" url="${org.jbpm.datasource.connection.url}" description="Raw Data Source" loginTimeout="0"
-              factory="${org.jbpm.datasource.driver.factory}"/>
+              user="${org.jbpm.datasource.username}" password="${org.jbpm.datasource.password}" url="${org.jbpm.datasource.connection.url}" URL="${org.jbpm.datasource.connection.url}" description="Raw Data Source" loginTimeout="0"
+              factory="${org.jbpm.datasource.driver.factory}" portNumber="${org.jbpm.datasource.port}" serverName="${org.jbpm.datasource.hostname}" databaseName="${org.jbpm.datasource.dbName}" testOnBorrow="false"
+              schema="${org.jbpm.persistence.schema}" driverType="4" />
```

Comment:
I don't 100% follow all these settings - I guess it is because of the other DB types. I wonder how setDriverType works on a Resource that doesn't have that option - perhaps Tomcat tests for each attribute before setting it or passes them as properties? I guess your testing showed that to be the case.